### PR TITLE
Improve README

### LINKED ISSUE #23: feature/improve-readme
README.md is out of date

### PR DESCRIPTION
**Pull request by quinnwai**
**Goal**: Make sure setup and README reflects project

I rebased on `features/packaging` just so everything was up to date, so you can ignore the first 6 commits.
Left some comments below on things that you had suggested in the README that I had questions about

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,81 @@
...
-## Testing
-```
+This Python project is designed to process Variant Call Format (VCF) files or other sources of variant information and perform lookup operations on Genomic Variation Representation Service (GA4GH VRS) identifiers. The GA4GH VRS identifiers provide a standardized way to represent genomic variations, making it easier to exchange and share genomic information.
+
+In addition, this project facilitates the retrieval of evidence associated with genomic alleles by leveraging the Genomic Data Representation and Knowledge Base (GA4GH MetaKB) service. GA4GH MetaKB provides a comprehensive knowledge base that links genomic variants to relevant evidence, enabling users to access valuable information about genomic alleles.
+
+## Features
+
+1. **VCF File Processing:**
+   - The project includes modules for reading and parsing VCF files, extracting relevant genomic information.
```
Comment (quinnwai) [inline README.md]: Not sure if you had a vision in mind for this, right now our VCF processing is limited and specific to 1000G and is really just calling bcftools. Let me know.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ Thank you for considering contributing to our Python project! By contributing, y
 
 ```bash
 git clone https://github.com/ohsu-comp-bio/vrs-python-testing
-python3 -m venv venv ; source venv/bin/activate
-pip install -r requirements.txt
+python3.10 -m venv venv; source venv/bin/activate
```
Comment (bwalsh) [inline CONTRIBUTING.md]: We shouldn't need to overspecify 3.10
We should be running on >=3.10

File: README.md (hunk 1 above)
Comment (bwalsh) [reply README.md]: Please suggest re-phrase, but don't get down to the level of specific dataset or tools

File: README.md [hunk 3]
```diff
@@ -1,37 +1,81 @@
...
+   ```bash
+   SEQREPO_ROOT=~
+   bash terra/setup.sh
+   ```
+
+### Usage
+
+**CLI**
+
+```bash
```
Comment (bwalsh) [inline README.md:61]: Lets update this based on testing.

**Review by bwalsh — CHANGES_REQUESTED** [review]
Generally looks good, however commits now in development still show up here.
Few minor comments.  Let's hold off until we test.

File: CONTRIBUTING.md (hunk 2 above)
Comment (quinnwai) [reply CONTRIBUTING.md]: updated

File: README.md (hunk 1 above)
Comment (quinnwai) [reply README.md]: did a small change, I think the modules was the only part that needed changing

**Pull request merged**